app/admin/RTMediaAdmin.php

```diff
@@ -272,34 +318,24 @@ public function rtmedia_premium_addon_notice() {
 			if ( ( ! $site_option || 'hide' !== $site_option ) ) {
 				rtmedia_update_site_option( 'rtmedia_premium_addon_notice', 'show' );
 				?>
-				<div class="updated rtmedia-pro-split-notice">
+				<div class="updated notice is-dismissible rtm-is-dismissible rtmedia-pro-split-notice" data-action="rtmedia_hide_premium_addon_notice" data-nonce="#rtcamp_pro_split">
+					<?php wp_nonce_field( 'rtcamp_pro_split', 'rtcamp_pro_split' ); ?>
```

Comment:
instead of passing nonce field can we please create nonce directly in attribute 
https://codex.wordpress.org/Function_Reference/wp_create_nonce
```
<div class="updated notice is-dismissible rtm-is-dismissible rtmedia-pro-split-notice" data-action="rtmedia_hide_premium_addon_notice" data-nonce="<?php echo esc_attr( wp_create_nonce( 'rtcamp_pro_split' ) ); ?>">
```